src/app/screens/History/components/NotesContainer/index.js

```diff
@@ -1,26 +1,37 @@
 import React from 'react';
-import { UTLabel } from '@widergy/energy-ui';
+import UTLabel from '@widergy/energy-ui/dist/components/UTLabel';
 import i18 from 'i18next';
-import { number, string, shape, arrayOf } from 'prop-types';
+import { number, string, shape, arrayOf, bool } from 'prop-types';
 
 import NoteItem from '../NoteItem';
 
-import { testNotes } from './utils';
 import styles from './style.module.scss';
 
-const NotesContainer = ({ notes = testNotes }) => (
-  <div className={styles.notesContainer}>
-    {notes?.length > 0 ? (
-      notes?.map(note => <NoteItem key={note.id} note={note} />)
-    ) : (
-      <UTLabel className={styles.notesContainerEmptyMsg} medium gray>
-        {i18.t('History:empty')}
-      </UTLabel>
-    )}
-  </div>
-);
+const NotesContainer = ({ notes }) =>
+  notes.notesLoading ? (
+    <div className={styles.spinnerContainer}>
```

Comment:
Not a requested change, but you can use UTLoading from energy UI! it receives loading={bool} as prop.